integration/kubernetes/e2e_conformance/setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Entry point to start a K8S cluster for end-to-end testing
+# If cluster is already running with Kata runtime class use run.sh
```

Comment:
If cluster is already running with Kata RuntimeClass, skip this script and use `run.sh`?